cs-property-service/src/main/java/org/egov/cpt/validator/PropertyValidator.java

```diff
@@ -1058,4 +1058,17 @@ private void validateDocumentsOnType(RequestInfo requestInfo, String tenantId, L
 				}
 			});
 	}
+
+	private void validateMortgageDocuments(MortgageRequest request, Map<String, String> errorMap) {
+		request.getMortgageApplications().forEach(mortgage -> {
+			this.validateDocumentsOnType(request.getRequestInfo(), mortgage.getTenantId(), mortgage.getApplicationDocuments(), errorMap, "MortgageRP");
```

Comment:
Find existing constants for this type or extract the text into a new constant.

Reply:
text extracted to new constant